tools/install.sh

```diff
@@ -149,10 +149,15 @@ chmod 770 ../smarty/templates_c
 
 # Changing group to 'www-data' or 'apache' to give permission to create directories in Document Repository module
 # Detecting distribution
-os_distro=$(hostnamectl |awk -F: '/Operating System:/{print $2}'|cut -f2 -d ' ')
+if ! os_distro=$(hostnamectl 2>/dev/null)
+then
+  os_distro="other"
```

Comment:
```suggestion
  os_distro="Other"
```

Otherwise the error message reads "other Linux distribution detected..." which looks a bit messy.